### PR TITLE
Add test for req-7

### PR DESCRIPTION
**Pull request by JasminMus**
**Pull Request Requirements:**

- Provide a brief description of the changeset.
- Include a screenshots if applicable
- Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

### CODE REVIEW

File: qms-tests/sanity/tests/documents/REQ-07.spec.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import { DocumentDetails, DocumentRights, DocumentStatus, NewDocument } from '../model/types'
+import { DocumentContentPage } from '../model/documents/document-content-page'
+import { prepareDocumentStep } from './common-documents-steps'
+import { DocumentHistoryPage } from '../model/documents/document-history-page'
+
+test.use({
+  storageState: PlatformSetting
+})
+
+test.describe('QMS. Documents tests', () => {
```
Comment (lexiv0re) [inline qms-tests/sanity/tests/documents/REQ-07.spec.ts]: Minor: can we be more specific here?

File: qms-tests/sanity/tests/documents/REQ-07.spec.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+  test.afterEach(async ({ browser }) => {
+    const contexts = browser.contexts()
+    for (const context of contexts) {
+      await context.close()
+    }
+  })
+
+  test('TESTS-384. Create a new doc version and previous one then marked to “Obsolete”', async ({ page, browser }) => {
+    await allure.description('Requirement\nUsers need to create a new document')
```
Comment (lexiv0re) [inline qms-tests/sanity/tests/documents/REQ-07.spec.ts]: Minor: ...to createa a new document _version_

**Review by lexiv0re — DISMISSED** [review]

File: qms-tests/sanity/tests/documents/REQ-07.spec.ts (hunk 1 above)
Comment (JasminMus) [reply qms-tests/sanity/tests/documents/REQ-07.spec.ts]: sure

File: qms-tests/sanity/tests/documents/REQ-07.spec.ts (hunk 2 above)
Comment (JasminMus) [reply qms-tests/sanity/tests/documents/REQ-07.spec.ts]: I usually put there what are requirements, creating a doc for example. But sure I can put a bit more context

**Review by lexiv0re — APPROVED** [review]

**Pull request merged**